cordova-plugin-outline/apple/src/OutlineSentryLogger.swift

```diff
@@ -110,11 +114,8 @@ class OutlineSentryLogger: DDAbstractLogger {
     dateFormatter.dateFormat = OutlineSentryLogger.kDateFormat
     // Breadcrumbs come from the application and VpnExtension processes. There is a limit to how
     // many we send, definied in OutlinePlugin.kMaxBreadcrumbs.
-    // Set the number of breadcrumbs for the VPN process to the maximum of an even split of the
-    // total breadcrumbs and the remaining breadcrumbs.
-    let maxNumVpnExtensionBreadcrumbs = max(
-        OutlinePlugin.kMaxBreadcrumbs / 2,
-        OutlinePlugin.kMaxBreadcrumbs - sentryClient.breadcrumbs.count());
+    // Set the number of breadcrumbs for the VPN process to half of the maximum breadcrumbs.
```

Comment:
Before we were able to determine the number of recorded breadcrumbs in order to make sure we fill the max breadcrumb quota with VPN extension logs. Since that's not possible anymore, we set at most half of the max breadcrumb quota to be VPN logs - the rest are already recorded app logs.